src/main/java/slimeknights/tconstruct/library/utils/ToolHelper.java

```diff
@@ -735,7 +738,7 @@ public static boolean attackEntity(ItemStack stack, ToolCore tool, EntityLivingB
         }
 
         // cooldown for non-projectiles
-        if(!isProjectile) {
+        if(!isProjectile && !skipCooldown) {
```

Comment:
Rename the parameter (and where it's called) to "startCooldown" or something similar, double negative is bad when thinking through the logic.